app/assets/stylesheets/views/_covid.scss

```diff
@@ -137,6 +137,47 @@ $covid-grey: #272828;
   margin-bottom: govuk-spacing(2);
 }
 
+.covid__video-placeholder {
+  position: relative;
+  padding: govuk-spacing(9) 0;
+  border: solid 2px govuk-colour("white");
+  text-align: center;
+
+  .govuk-body {
+    margin: 0;
+  }
+
+  .covid__bold {
+    font-weight: bold;
+  }
+}
+
+.covid__video-logo {
+  @include govuk-font(19, $weight: bold, $line-height: 1);
+  position: absolute;
+  top: 0;
+  left: 0;
+  padding: govuk-spacing(1);
+  padding-top: 9px;
```

Comment:
Rather than 9px - why don't we just use `govuk-spacing(2)` which gives 10

Reply:
```suggestion
  padding-top: govuk-spacing(2);
```

Reply:
Changed.